makefiles/buildtests.inc.mk

```diff
@@ -38,7 +38,7 @@ buildtest:
 	@ \
 	BUILDTESTOK=true; \
 	APP_RETRY=0; \
-	for BOARD in $$($(MAKE) -s info-boards-supported); do \
+	for BOARD in ${BOARDS}; do \
```

Comment:
Mhh... I'm not sure where the `BOARDS` comes from. But `make -s info-boards-supported` calculates this... if this is done in another way, we might be able to remove or simplify the `info-boards-supported` target (but not in this PR)